client/jetpack-cloud/sections/partner-portal/primary/wpcom-atomic-hosting/card-content.tsx

```diff
@@ -63,12 +62,23 @@ export default function CardContent( {
 		}
 	};
 
-	const getPreviousProductName = ( planSlug: string ) => {
+	const getFeaturesHeading = ( planSlug: string ) => {
 		switch ( planSlug ) {
 			case PLAN_BUSINESS:
-				return 'Premium';
+				return translate( 'Everything in {{a}}Premium{{/a}}, plus:', {
+					components: {
+						a: (
+							// For Business plan, we want to redirect user to find out more about features included in the  Premium plan.
+							<a
+								href="https://wordpress.com/pricing/#lpc-pricing"
+								target="_blank"
+								rel="noreferrer"
+							/>
+						),
+					},
+				} );
 			case PLAN_ECOMMERCE:
-				return 'Business';
+				return translate( 'Everything in Business, plus:' );
```

Comment:
I Simplified the translation and opted for a non-parameterized translation string.